style.css

```diff
@@ -63,6 +95,229 @@ h1 {
   border: 2px solid #dfe1e6;
 }
 
+#work-section h2 {
+  font-size: 40px;
+  text-align: center;
+  color: #172b4d;
+  padding: 30px;
+}
+
+#card-container {
+  width: 100%;
+  display: grid;
+  justify-content: center;
+  padding: 10px;
+  gap: 15px;
+  grid-template-columns: repeat(auto-fill, 327px);
+}
+
+#about-me-text {
+  background-image: url('./images/about-me-mobile.png');
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px 0 20px 0;
+  gap: 20px;
+  background-size: cover;
+  background-position: 100%;
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+#about-me-text h2 {
+  font-size: 40px;
+  font-weight: bold;
+  color: #172b4d;
+}
+
+#about-me-text p {
+  font-size: 20px;
+  font-weight: 500;
+  color: #42526e;
+  padding: 0 20px 0 20px;
+  text-align: center;
+}
+
+#about-me-text button {
+  background-color: #36b37f;
+  color: white;
+  font-size: 20px;
+  outline-width: 0;
+  font-weight: 400;
+  align-self: center;
+  padding: 15px;
+  border-radius: 10px;
+  border: 0;
+}
+
+#about-me-skills {
+  width: 100%;
+  display: grid;
+  justify-content: center;
+  padding: 10px;
+  gap: 15px;
+  grid-template-columns: repeat(auto-fill, 327px);
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` and etc. which makes your design `inconsistent` and looks unprofessional. You must need to make the design as close as possible to the `Figma` design because it will make you a better developer who always implements the details which results in *a good user experience*. Please have a look at the below-attached image and table. Please don't add any custom data at this moment.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-04 15-06-24](https://user-images.githubusercontent.com/66476329/236191825-9b976a89-959d-4bcd-939b-4f1050bf0730.png)

</td>
<td>

![Screenshot from 2023-05-04 15-06-11](https://user-images.githubusercontent.com/66476329/236191845-8f6335b2-8383-4823-a05d-86c0cec8ee61.png)

</td>
</tr>
</table>